modules/031-linstor/docs/USAGE.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "The linstor module: configuration examples"
+---
+
+We're working hard to make Deckhouse easy to use and don't want to overwhelm you with too much information. Therefore, we decided to provide you with a simple and familiar interface for configuring LINSTOR. Namely, all you need is to create LVM partitions with certain tags. Such partitions will be automatically added to LINSTOR and will be available for use as a StorageClass in Kubernetes.
```

Comment:
> There is no information about "specific tags" in ADVANCED_USAGE.md

This service is not implemented yet, please see
- https://github.com/deckhouse/deckhouse/issues/903

The main point to hide complexity of linstor cli under **Advanced usage**.
While **Usage** will contain only information about this simplified solution. 

> "LVM partition" - is it a "logical volume"

It can be volume group or thin pool, fixed

Reply:
"This service" was confusing, I thought "service" is equal to the whole module. I've added suggestions in the new review.